querybook/server/datasources/query_execution.py

```diff
@@ -779,3 +754,39 @@ def transpile_query(
 ):
     transpiler = get_transpiler_by_name(transpiler_name)
     return transpiler.transpile(query, from_language, to_language)
+
+
+@register("/query_execution/<int:query_execution_id>/approve_review/", methods=["PUT"])
+def approve_query_review(query_execution_id):
+    with DBSession() as session:
+        verify_query_execution_permission(query_execution_id)
+        reviewer_id = current_user.id
+        query_execution = logic.get_query_execution_by_id(
+            query_execution_id, session=session
+        )
+        query_review = query_execution.review
+        api_assert(
+            query_review is not None,
+            "Review not found",
+        )
+        query_review_handler.approve_review(query_review.id, reviewer_id, session)
+    return query_execution.to_dict(with_statement=False, with_query_review=True)
+
+
+@register("/query_execution/<int:query_execution_id>/reject_review/", methods=["PUT"])
+def reject_query_review(query_execution_id, rejection_reason):
+    with DBSession() as session:
+        verify_query_execution_permission(query_execution_id)
+        reviewer_id = current_user.id
+        query_execution = logic.get_query_execution_by_id(
+            query_execution_id, session=session
+        )
+        query_review = query_execution.review
+        api_assert(
+            query_review is not None,
+            "Review not found",
+        )
```

Comment:
this part is same as above in approve endpoint,  can it be moved to the query view handler and make those check shareable? 
also here you're getting the execution by id, and in query_review_handler.approve/reject, it is getting the review by review_id, seems it can be consolidated.